sequencing/vcf_pca/dxapp.json

```diff
@@ -171,16 +173,26 @@
     }
   ],
   "runSpec": {
-    "execDepends": [
-      {"name": "libgsl0ldbl"},
-      {"name": "pigz"},
-      {"name": "tabix"}
-    ],
     "interpreter": "bash",
-    "file": "src/vcf_pca.sh"
+    "file": "src/vcf_pca.sh",
+    "restartableEntryPoints": "all", 
+    "distribution": "Ubuntu",
+    "release": "14.04",
+    "timeoutPolicy": {
+      "*": { "hours": 120 }
+    }
+  },
+  "regionalOptions": {
+    "aws:us-east-1": {"systemRequirements": {"*": {"instanceType": "mem1_ssd1_x16"}},
+                    "assetDepends": [{"id": "record-FGpf7zQ0kfbZxj6q4ZzQ2BPG"}],
+                    "resources" : "project-FGqF5B00ByvPXG4G87yQy23G"},
+    "azure:westus": {"systemRequirements": {"*": {"instanceType": "azure:mem2_ssd1_x16"}},
+                    "assetDepends": [{"id": "record-FGpfZ1j95qzXV43bP5y5JKgV"}],
+                    "resources" : "project-FGvp3009BJq40pkZ683v3754"}
```

Comment:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz